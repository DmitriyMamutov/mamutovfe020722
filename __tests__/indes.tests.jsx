import { render, screen } from "@testing-library/react";
import Home from "@/pages/index";

describe("Home", () => {
  it("renders a heading", () => {
    render(<Home />);

    const heading = screen.getByText(/header/i);

    expect(heading).toBeInTheDocument();
  });

  it("snapshot", () => {
    const home = render(<Home />);

    expect(home).toMatchSnapshot();
  });
});
