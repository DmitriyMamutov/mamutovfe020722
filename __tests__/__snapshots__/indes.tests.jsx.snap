// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="header"
        >
          <div
            class="container"
          >
            <div
              class="header__text"
            >
              Header
            </div>
          </div>
        </div>
        <div
          class="images"
        >
          <div
            class="container"
          >
            <div
              class="images-list"
            >
              <div
                class="images-list-item"
              >
                <div
                  class="images-list-item__image"
                >
                  <img
                    src="/static/images/main/cats.jpg"
                  />
                </div>
                <button
                  class="button button button--large"
                  style="background: rgb(243, 245, 250);"
                >
                  Click
                </button>
              </div>
              <div
                class="images-list-item"
              >
                <div
                  class="images-list-item__image"
                >
                  <img
                    src="/static/images/main/chubaka.jpg"
                  />
                </div>
                <button
                  class="button button button--large"
                  style="background: rgb(243, 245, 250);"
                >
                  Click
                </button>
              </div>
              <div
                class="images-list-item"
              >
                <div
                  class="images-list-item__image"
                >
                  <img
                    src="/static/images/main/dogs.jpg"
                  />
                </div>
                <button
                  class="button button button--large"
                  style="background: rgb(243, 245, 250);"
                >
                  Click
                </button>
              </div>
              <div
                class="images-list-item"
              >
                <div
                  class="images-list-item__image"
                >
                  <img
                    src="/static/images/main/dyno.jpg"
                  />
                </div>
                <button
                  class="button button button--large"
                  style="background: rgb(243, 245, 250);"
                >
                  Click
                </button>
              </div>
              <div
                class="images-list-item"
              >
                <div
                  class="images-list-item__image"
                >
                  <img
                    src="/static/images/main/planets.jpg"
                  />
                </div>
                <button
                  class="button button button--large"
                  style="background: rgb(243, 245, 250);"
                >
                  Click
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="footer"
        >
          <div
            class="container"
          >
            <div
              class="footer__text"
            >
              Footer
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="header"
      >
        <div
          class="container"
        >
          <div
            class="header__text"
          >
            Header
          </div>
        </div>
      </div>
      <div
        class="images"
      >
        <div
          class="container"
        >
          <div
            class="images-list"
          >
            <div
              class="images-list-item"
            >
              <div
                class="images-list-item__image"
              >
                <img
                  src="/static/images/main/cats.jpg"
                />
              </div>
              <button
                class="button button button--large"
                style="background: rgb(243, 245, 250);"
              >
                Click
              </button>
            </div>
            <div
              class="images-list-item"
            >
              <div
                class="images-list-item__image"
              >
                <img
                  src="/static/images/main/chubaka.jpg"
                />
              </div>
              <button
                class="button button button--large"
                style="background: rgb(243, 245, 250);"
              >
                Click
              </button>
            </div>
            <div
              class="images-list-item"
            >
              <div
                class="images-list-item__image"
              >
                <img
                  src="/static/images/main/dogs.jpg"
                />
              </div>
              <button
                class="button button button--large"
                style="background: rgb(243, 245, 250);"
              >
                Click
              </button>
            </div>
            <div
              class="images-list-item"
            >
              <div
                class="images-list-item__image"
              >
                <img
                  src="/static/images/main/dyno.jpg"
                />
              </div>
              <button
                class="button button button--large"
                style="background: rgb(243, 245, 250);"
              >
                Click
              </button>
            </div>
            <div
              class="images-list-item"
            >
              <div
                class="images-list-item__image"
              >
                <img
                  src="/static/images/main/planets.jpg"
                />
              </div>
              <button
                class="button button button--large"
                style="background: rgb(243, 245, 250);"
              >
                Click
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="footer"
      >
        <div
          class="container"
        >
          <div
            class="footer__text"
          >
            Footer
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
